.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+indent_style = space
+
+[*.{md}]
+indent_size = 4
+
+[*.{ex,exs}]
+indent_size = 2
```

Comment:
Is this the standard? In Erlang it is 4.

Reply:
Yes; if you `mix new indentation-test-project` you'll see `mix.exs` _et al_ with an indentation of 2, and `mix format` will change it back to 2 if you try to manually make it 4.